solc/CommandLineInterface.cpp

```diff
@@ -510,8 +519,13 @@ void CommandLineInterface::createFile(string const& _fileName, string const& _da
 bool CommandLineInterface::parseArguments(int _argc, char** _argv)
 {
 	// Declare the supported options.
-	po::options_description desc(
-		R"(solc, the Solidity commandline compiler.
+	po::options_description desc(R"(solc, the Solidity commandline compiler.
+
+Solidity Copyright (C) 2014-2017 Solidity contributors.
```

Comment:
I think we should sort this in a separate PR which also adds it to the files. Gives us more time to find the proper wording :)